.pre-commit-config.yaml

```diff
@@ -7,15 +7,15 @@ repos:
   - id: check-yaml
   - id: end-of-file-fixer
   - id: trailing-whitespace
-- repo: https://github.com/markdownlint/markdownlint.git
-  rev: v0.13.0
-  hooks:
-  - id: markdownlint_docker
-  name: Markdownlint Docker
-  description: Run markdown lint on your Markdown files using the project docker image
-  language: docker_image
-  files: \.(md|mdown|markdown)$
-  entry: markdownlint/markdownlint
+#- repo: https://github.com/markdownlint/markdownlint.git
```

Comment:
I'm not sure why the image pull fails. But, Rather than disable what about changing this to the [same as what's in Kepler](https://github.com/sustainable-computing-io/kepler/pull/1555/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R19)

Note that the following file is also needed: https://github.com/sustainable-computing-io/kepler/pull/1555/files#diff-7acf5910ecddb0a88c58debaedb129a30ebad965e878f6ebba3dad5a4fd8758a

Or I can create a separate PR... WDYT?

Reply:
I created a PR https://github.com/sustainable-computing-io/local-dev-cluster/pull/91